src/pages/Article.tsx

```
@@ -230,18 +183,18 @@ const Article = () => {
                   aria-label="Bookmark article"
                 >
                   <Bookmark size={18} />
                 </button>
-              </div>
+              </div> */}
               
               {/* Content */}
               <div 
                 className="prose prose-lg max-w-none"
-                dangerouslySetInnerHTML={{ __html: articleContent }}
+                dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(article.content || 'Chưa có nội dung') }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)